--- a/src/smartdns.c
+++ b/src/smartdns.c
@@ -47,7 +47,7 @@
 #define RESOLVE_FILE "/etc/resolv.conf"
 #define MAX_LINE_LEN 1024
 #define MAX_KEY_LEN 64
-#define SMARTDNS_PID_FILE "/var/run/smartdns.pid"
+#define SMARTDNS_PID_FILE "/opt/var/run/smartdns.pid"
 #define TMP_BUFF_LEN_32 32
 
 static int verbose_screen;
--- a/src/dns_conf.h
+++ b/src/dns_conf.h
@@ -48,8 +48,8 @@ extern "C" {
#define DEFAULT_DNS_TLS_PORT 853
#define DEFAULT_DNS_HTTPS_PORT 443
#define DNS_MAX_CONF_CNAME_LEN 256
-#define SMARTDNS_CONF_FILE "/etc/smartdns/smartdns.conf"
-#define SMARTDNS_LOG_FILE "/var/log/smartdns/smartdns.log"
-#define SMARTDNS_AUDIT_FILE "/var/log/smartdns/smartdns-audit.log"
-#define SMARTDNS_CACHE_FILE "/tmp/smartdns.cache"
-#define SMARTDNS_DEBUG_DIR "/tmp/smartdns"
+#define SMARTDNS_CONF_FILE "/opt/etc/smartdns/smartdns.conf"
+#define SMARTDNS_LOG_FILE "/opt/var/log/smartdns.log"
+#define SMARTDNS_AUDIT_FILE "/opt/var/log/smartdns-audit.log"
+#define SMARTDNS_CACHE_FILE "/opt/tmp/smartdns.cache"
+#define SMARTDNS_DEBUG_DIR "/opt/tmp/smartdns"

