--- a/etc/smartdns/smartdns.conf	2023-03-14 10:59:57.606454800 +0800
+++ b/etc/smartdns/smartdns.conf	2023-03-14 11:17:43.566189638 +0800
@@ -147,7 +147,7 @@
 # log-num: number of logs, 0 means disable log
 log-level info
 
-# log-file /var/log/smartdns/smartdns.log
+# log-file /opt/var/log/smartdns/smartdns.log
 # log-size 128k
 # log-num 2
 # log-file-mode [mode]: file mode of log file.
@@ -157,7 +157,7 @@
 # audit-enable yes
 # audit-SOA [yes|no]: enable or disable log soa result.
 # audit-size size of each audit file, support k,m,g
-# audit-file /var/log/smartdns-audit.log
+# audit-file /opt/var/log/smartdns-audit.log
 # audit-console [yes|no]: output audit log to console.
 # audit-file-mode [mode]: file mode of audit file.
 # audit-size 128k